### DISCUSSION TITLE
New Wiki page

### DISCUSSION THREAD

**Discussion by serych** [EngineerGuy314/pico-WSPRer#33] [Show and tell]
Hi Rob!
Unfortunately wiki pages of the repository are not synchronised to the forks, but I have found the trick how to do it:
https://gist.github.com/omaraboumrad/35654da0a376c57a2e0ab4d92ad0c339 , synchronised your wiki to my fork and created the new page "How does the pico oscilator work" where I tried to describe, how the pico-WSPRer code works:
https://github.com/serych/pico-WSPRer/wiki/how-does-pico-oscilator-work 
It seems to me, that it is little bit trickier to merge the changes to your repository (it is described in the section "Core Contributors" of the  omaraboumrad article). But if you like my article, you could probably just copy it from my fork and paste it to your wiki as a new article using Ctrl-C Ctrl-V. May be it could be easier than the omaraboumrads solution.
Jakub, OK1SE

**Comment by EngineerGuy314** [comment]
Hi Jakub,
That looks great! I think I will pick a route halfway in between a merge and a copy paste: I will put a section in the wiki with essentially just a link to your wiki. I think that should work good, and changes you make on your end will be available. 

btw the TELEN stuff can now be configured from the user menu. I hope to get a test flight in the air thursday.
    On Monday, July 29, 2024 at 06:13:33 PM EDT, Jakub Šerých ***@***.***> wrote:  
 
 


Hi Rob!
Unfortunately wiki pages of the repository are not synchronised to the forks, but I have found the trick how to do it:
https://gist.github.com/omaraboumrad/35654da0a376c57a2e0ab4d92ad0c339 , synchronised your wiki to my fork and created the new page "How does the pico oscilator work" where I tried to describe, how the pico-WSPRer code works:
https://github.com/serych/pico-WSPRer/wiki/how-does-pico-oscilator-work
It seems to me, that it is little bit trickier to merge the changes to your repository (it is described in the section "Core Contributors" of the omaraboumrad article). But if you like my article, you could probably just copy it from my fork and paste it to your wiki as a new article using Ctrl-C Ctrl-V. May be it could be easier than the omaraboumrads solution.
Jakub, OK1SE
